### DISCUSSION TITLE
Turn Page feature?

### DISCUSSION THREAD

**Discussion by DarkJester89** [lnreader/lnreader#516] [Ideas]
I'm coming to LNReader after years of reading ebooks on ReaderEra and comicbooks on Perfect Viewer, is it possible to put a page flipping feature in the reading function, where instead of a unless scroll, you can swipe to go page to page? Thanks for your time!

**Comment by rajarsheechatterjee** [comment]
Create an [Feature Request](https://github.com/LNReader/lnreader/issues/new?assignees=&labels=Feature+Request&projects=&template=request_feature.yml) if there isn't an existing issue for it.
